### PR TITLE
Add slides for sqlflow offline meetup

### PR DESCRIPTION
**Pull request by tonyyang-svail**
To run the presentation. `cd cmd/demo/slides/201905/ && present`.

### CODE REVIEW

**Review by weiguoz — DISMISSED** [review]
LGTM

I would suggest add `.DS_Store` to `.gitignore`.

File: cmd/demo/slides/201905/sqlflow.slide [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+- TensorFlow, XGboost, ...
+
+It extends the SQL syntax to enable model training and inference.
+
+* Syntax: Regular SQL
+.code regular.sql
+
+* Syntax: Train
+.code train.sql
```
Comment (wangkuiyi) [inline cmd/demo/slides/201905/sqlflow.slide:24]: Just being curious -- can we actually run train.sql from within the Go slide?

**Review by wangkuiyi — COMMENTED** [review]
I didn't have time to go over the content of the slide.

It doesn't look alright to put slides and a text-mode client in a directory named `demo`.

Maybe the slide goes into `doc/` and the client goes into `cmd/sqlflowcli`.

File: cmd/demo/slides/201905/sqlflow.slide (hunk 1 above)
Comment (tonyyang-svail) [reply cmd/demo/slides/201905/sqlflow.slide:24]: I don't think so.

**Comment by tonyyang-svail** [comment]
> It doesn't look alright to put slides and a text-mode client in a directory named demo.
> Maybe the slide goes into doc/ and the client goes into cmd/sqlflowcli.

@wangkuiyi Will do in the next PR.

**Review by weiguoz — APPROVED** [review]

**Pull request merged**